utils/land-patterns.stanza

```diff
@@ -617,18 +691,19 @@ public pcb-landpattern dfn-landpattern ( n-pad:Int, pad-pitch:Double, pad-dims:[
   make-dfn-landpattern(n-pad,pad-pitch,pad-dims,pad-x0,body-dims)
   pad p[n-pad + 1] : smd-pad(hs[0],hs[1]) at loc(0.0, 0.0)  
 
-
-;TODO: Until added to core
-; defn name! (r:Ref) :
-  ; name(r as VarRef)
-
-public defn red-alph-letter (idx:Int) : ["A" "B" "C" "D" "E" "F" "G" "H" "J" "K" "L" "M" "N" "P" "R" "T" "U" "V" "W" "Y" 
-   "AA" "AB" "AC" "AD" "AE" "AF" "AG" "AH" "AJ" "AK" "AL" "AM" "AN" "AP" "AR" "AT" "AU" "AV" "AW" "AY"  
-   "BA" "BB" "BC" "BD" "BE" "BF" "BG" "BH" "BJ" "BK" "BL" "BM" "BN" "BP" "BR" "BT" "BU" "BV" "BW" "BY"][idx]
+public defn red-alph-letter (idx:Int):
+  val table = "ABCDEFGHJKLMNPRTUVWY"
+  val buf = StringBuffer()
+  var rem = idx
+  let loop (): 
+    print(buf, table[rem % length(table)])
+    if rem > 0:
```

Comment:
not needed

Reply:
good catch, the loop was totally wrong. Refactored and added a test!